CRM/Contribute/Form/Task/PDFLetterCommon.php

```diff
@@ -335,9 +335,6 @@ public static function combineContributions($existing, $contribution, $separator
    * @param int $groupByID
    */
   public static function assignCombinedContributionValues($contact, $contributions, $groupBy, $groupByID) {
-    if (!defined('CIVICRM_MAIL_SMARTY') || !CIVICRM_MAIL_SMARTY) {
-      return;
-    }
     CRM_Core_Smarty::singleton()->assign('contact_aggregate', $contact['contact_aggregate']);
```

Comment:
This early return actually adds no value. No performance benefit and it makes it harder for extensions & unit tests to interact